code/lib/core-server/src/presets/common-preset.ts

```diff
@@ -185,15 +185,21 @@ export const previewAnnotations = async (base: any, options: Options) => {
   return [...config, ...base];
 };
 
+const fastBuildFeatures = (value: boolean) => ({
+  fastBuildEmptyBlocks: value,
+});
+
 export const features = async (
-  existing: StorybookConfig['features']
+  existing: StorybookConfig['features'],
+  options: Options
 ): Promise<StorybookConfig['features']> => ({
   ...existing,
   warnOnLegacyHierarchySeparator: true,
   buildStoriesJson: false,
   storyStoreV7: true,
   argTypeTargetsV7: true,
   legacyDecoratorFileOrder: false,
+  ...fastBuildFeatures(!!options.test),
```

Comment:
@kasperpeulen this preset property is invoked very very early on, and in fact placed on a global in nodeJS here: 
https://github.com/storybookjs/storybook/blob/c05b719a9d03291a25ff953cfbfbafe7d1166830/code/lib/core-server/src/build-static.ts#L146

But the better way to access it is via `options`, as it's set here:
https://github.com/storybookjs/storybook/blob/c05b719a9d03291a25ff953cfbfbafe7d1166830/code/lib/core-server/src/build-static.ts#L134-L138

That's how I used it in this PR, it's pretty convenient..